clusterloader2/pkg/framework/client/objects.go

```diff
@@ -46,6 +46,9 @@ const (
 	// Parameters for namespace deletion operations.
 	defaultNamespaceDeletionTimeout  = 10 * time.Minute
 	defaultNamespaceDeletionInterval = 5 * time.Second
+
+	namespace       = "monitoring"
```

Comment:
Please name it in more descriptive way, e.g. monitoringNamespace

Reply:
Actually, let's remove it completely from this file - it shouldn't live here (see also my comment below).

Reply:
@wojtek-t updated!